common/include/fwd-event.h

```diff
@@ -7,6 +7,8 @@
 
 #pragma once
 
+#include "pstypes.h"
```

Comment:
I would prefer to use the standard header (`<cstdint>`) here, since nothing else from `pstypes.h` is necessary.